frontend/vite.config.ts

```diff
@@ -1,12 +1,12 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig } from "vite"
+import react from "@vitejs/plugin-react"
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
   server: {
     proxy :{
-      "/api" : {
+      "/todo" : {
```

Comment:
Not totally sure but I kind of remember that the usage of api is the "better" choice?